workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_home_resources.scss

```diff
@@ -0,0 +1,101 @@
+.home-resources {
+	display: grid;
+	gap: 48px;
+	grid-template-columns: repeat(auto-fit, 184px);
```

Comment:
Do we need to use display: grid? I'd prefer to just use the bootstrap markup like in the learn repo and then we wouldn't need most of this extra css. 

i.e. like 


```
 <div class="col-md-12 row">
                    <div class="col-lg-4">
                   ...
                    </div>

                    <div class="col-lg-4">
                    ...
                    </div>
```

Reply:
I replaced the grid CSS with bootstrap grid.